stable/artifactory-ha/README.md

```diff
@@ -780,6 +781,7 @@ The following table lists the configurable parameters of the artifactory chart a
 | `artifactory.primary.javaOpts.jmx.ssl`              | Enable SSL           |  `false` |
 | `artifactory.primary.javaOpts.other`            | Artifactory primary node additional java options |                     |
 | `artifactory.primary.persistence.existingClaim` | Whether to use an existing pvc for the primary node | `false`            |
+| `artifactory.node.labels`                       | Artifactory primary node labels                  | `{}`                |
```

Comment:
This should say artifactory member nodes labels

Reply:
Resolved